content/md/first-steps/manage-your-attributes.md

```diff
@@ -92,15 +92,16 @@ The attribute is now created, you **can add a translation for each enabled local
 
 ::: warning
 Please note that the **following attribute codes cannot be used** in Akeneo PIM:    
-`id`, `associationTypes`, `category`, `categories`, `categoryId`, `completeness`, `enabled`, `family`, `Family`, `groups`, `associations`, `products`, `scope`, `treeId`, `values`, `label`, `parent`.    
-We strongly recommend you to do not use any of these codes even if the letter case is different, for instance `Id`, `Groups`.
+`id`, `associationTypes`, `category`, `categories`, `categoryId`, `completeness`, `enabled`, `family`, `groups`, `associations`, `products`, `scope`, `treeId`, `values`, `label`, `parent`
```

Comment:
Please keep "Family" and "family", it's not the same. Both are forbidden.

Reply:
Sorry Laure! I didn't understand what happened, the entire line was deleted in my editor, I thought I did a mistake so I copied pasted the current helpcenter... Sorry if you already fixed it! I change it ;) thanks!